app/components/primer/avatar_stack_component.rb

```diff
@@ -40,14 +43,17 @@ class AvatarStackComponent < Primer::Component
     # @param align [Symbol] <%= one_of(Primer::AvatarStackComponent::ALIGN_OPTIONS) %>
     # @param tooltipped [Boolean] Whether to add a tooltip to the stack or not.
     # @param body_arguments [Hash] Parameters to add to the Body. If `tooltipped` is set, has the same arguments as <%= link_to_component(Primer::Tooltip) %>.
+    #   The default tag is <%= pretty_value(Primer::AvatarStackComponent::DEFAULT_BODY_TAG) %> but can be changed using `tag:`
+    #   to <%= one_of(Primer::AvatarStackComponent::BODY_TAG_OPTIONS).downcase %>
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(tag: DEFAULT_TAG, align: ALIGN_DEFAULT, tooltipped: false, body_arguments: {}, **system_arguments)
       @align = fetch_or_fallback(ALIGN_OPTIONS, align, ALIGN_DEFAULT)
       @system_arguments = system_arguments
       @tooltipped = tooltipped
       @body_arguments = body_arguments
 
-      @body_arguments[:tag] ||= :div # rubocop:disable Primer/NoTagMemoize
+      body_tag = @body_arguments[:tag] || DEFAULT_BODY_TAG
```

Comment:
🤔 Why do we need this line? Isn't fetch_or_fallback meant to handle this behavior?

Reply:
This is definitely a little awkward but necessary in this scenario with `body_arguments`. 

`fetch_or_fallback` only falls back in prod and will raises an error otherwise.

If `@body_arguments[:tag]` is nil, the `fetch_or_fallback` method will raise in an error and cause our development tests to fail with:

```
Expected one of: [:div, :span]
Got: nil
```

Alternatively, I considered setting the tag default within the hash like:

```
def initialize(...body_arguments: { tag: DEFAULT_BODY_TAG}, **system_arguments)
```

However, it seemed like the default would easily get lost when a user passes in their own `body_arguments` for any other attribute.

Reply:
@khiga8 ah, that's helpful context. Do you think this might be a sign that `body` should be a slot?

Reply:
I was wondering that too. @manuelpuyol, would you have some context of why `body_arguments` over having a `body` slot?

Reply:
We chose to use arguments instead of slots here because customizing `body` was rare, so having a slot for the edge case would make it more complicated for eveyone using it:

```
render Primer::AvatarStack.new do |c|
  c.body do |b|
    b.avatar
    b.avatar
...
```

Reply:
ah, gotcha! It is unexpected to me that the avatar stack is composed with an extra element like:
```
<div class="AvatarStack AvatarStack--three-plus">
  <div class="AvatarStack-body"> 
     <img>
     <img>
 ```
vs

```
<div class="AvatarStack AvatarStack--three-plus">
   <img>
   <img>
``` 
It looks like there's some [historical context][0] and a css reason for why it's like this.

I see what you're saying @manuelpuyol that maybe the consumer doesn't need to know exactly how this component is composed. Doing a quick grep for`AvatarStack-body` in dotcom, I see several instances with a tooltip and `aria-label` set on the `body` which makes me think it *could* be worth exposing `body` as a slot.  

I think we should revisit this API question outside of this PR! What do you think of moving forward with this tag change for now? @joelhawksley 

[0]: https://primer.style/css/components/avatars#avatar-stack